message PackedFields {
  repeated int32 f = 6 [packed=true];
}